pvr.dvbviewer/addon.xml.in

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.dvbviewer"
-  version="2.0.1"
+  version="3.1.0"
```

Comment:
arr my fault again. I thought 2.1.0 but typed 3.1.0.. I guess it's time for some break.